pkg/restclient/volume.go

```diff
@@ -95,3 +95,9 @@ func (c *Client) OptionGroupDelete(group string) error {
 	url := fmt.Sprintf("/v1/volumes/options-group/%s", group)
 	return c.del(url, nil, http.StatusOK, nil)
 }
+
```

Comment:
Have these functions in a separate file in the same directory(`quota.go`)

Reply:
done